WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt

```diff
@@ -318,7 +318,13 @@ class SitePreviewViewModel @Inject constructor(
         object SitePreviewFullscreenProgressUiState : SitePreviewUiState(
                 fullscreenProgressLayoutVisibility = true
         ) {
-            const val loadingTextResId = R.string.notification_new_site_creation_creating_site_subtitle
+            val loadingTextResId: List<Int> = listOf(
```

Comment:
Replace the single-valued res-id on hte view-model with an ordered collection of the texts to be displayed